File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/CatastoCampo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class CatastoCampo
    {

        public double area { get; set; }

        public ParticelleCatastali particella { get; set; }

        public bool flag_cancellazione { get; set; }

        public CatastoCampo()
        {
            flag_cancellazione = false;
        }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/CatastoCampo.ts
import { ParticelleCatastali, PKParticelleCatastali } from './ParticelleCatastali';

export class CatastoCampo {
    area: number;
    superficieCondotta: number;
    particella: ParticelleCatastali;
    flag_cancellazione: boolean;

    constructor() {
        this.flag_cancellazione = false;
    }

}

